recipes/cms/meta.yaml

```diff
@@ -0,0 +1,124 @@
+package:
+  name: cms
+  version: "2.0.0b"
+
+about:
+  home: http://broad-cms.readthedocs.io/en/latest/
+  license: GPLv3
+  summary: "CMS (composite of multiple signals) is a suite of tools for resolving signatures of recent positive selection in population genetic data."
+
+source:
+  fn: cms-2.0.0-beta.tar.gz
+  url: https://github.com/broadinstitute/cms/archive/2.0.0-beta.tar.gz
+  md5: 3e5754bf0a6eeb71cc483dcdae10d8c0
+
+build:
+  number: 0
+  rpaths:
+      - lib/
+      - lib//
+
+requirements:
+  build:
```

Comment:
The bioconda recommendations have changed for compiler selectors. gcc and llvm should now have the commented-out selectors (I need to update the viral-ngs recipe...):

```yaml 
requirements:
  build:
    - gcc   # [not osx]
    - llvm  # [osx]

  run:
    - libgcc    # [not osx]
```